webapp/src/components/data_selectors/jira_commentvisibility_selector/jira_commentvisibility_selector.tsx

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import {ReactSelectOption} from 'types/model';
+
+import BackendSelector, {Props as BackendSelectorProps} from '../backend_selector';
+
+const stripHTML = (text: string) => {
+    if (!text) {
+        return text;
+    }
+
+    const doc = new DOMParser().parseFromString(text, 'text/html');
+    return doc.body.textContent || '';
+};
+
+type Props = BackendSelectorProps & {
+    searchCommentVisibilityFields: (params: {fieldValue: string}) => (
+        Promise<{data: {groups: {items: {name: string}[]}}; error?: Error}>
+    );
+    fieldName: string;
+};
+
+export default class JiraCommentVisibilitySelector extends React.PureComponent<Props> {
+    fetchInitialSelectedValues = async (): Promise<ReactSelectOption[]> => {
+        if (!this.props.value || (this.props.isMulti && !this.props.value.length)) {
+            return [];
+        }
+
+        return this.searchCommentVisibilityFields('');
+    };
+
+    searchCommentVisibilityFields = (inputValue: string): Promise<ReactSelectOption[]> => {
+        const {instanceID} = this.props;
+        const params = {
+            fieldValue: inputValue,
+            instance_id: instanceID,
```

Comment:
if we are using instanceID only once there is no need to create an extra variable 
```suggestion
            instance_id:this.props.instanceID,
```